project.nix

```diff
@@ -1,211 +1,120 @@
-{ system, withHoogle }:
-(import ./dep/reflex-platform { inherit system; }).project ({ pkgs, hackGet, ... }: 
-let 
-  gitignoreSrc = pkgs.fetchFromGitHub { 
-    owner = "hercules-ci";
-    repo = "gitignore";
-    # put the latest commit sha of gitignore Nix library here:
-    rev = "00b237f";
-    # use what nix suggests in the mismatch message here:
-    sha256 = "sha256:186pvp1y5fid8mm8c7ycjzwzhv7i6s3hh33rbi05ggrs7r3as3yy";
+let nixpkgsRev = "dc80d7b";
+in {
+  pkgs ? import (builtins.fetchTarball "https://github.com/nixos/nixpkgs/archive/${nixpkgsRev}.tar.gz") { },
+  # Cabal project name
+  name ? "neuron",
+  compiler ? pkgs.haskell.packages.ghc883,
```

Comment:
Any reason why we are not using the default compiler (`pkgs.haskellPackages`)? Because then you can take advantage of nixos cache for Haskell packages.

Reply:
It is the default compiler. I just used that because that's what I found. I can gladly replace it with pkgs.haskellPackages, though I don‘t expect that will change anything regarding the caching.

Reply:
Yes, let's just use `pkgs.haskellPackages`.